Files/Filesystem/Drives.cs

```diff
@@ -42,6 +40,8 @@ public bool ShowUserConsentOnInit
         public DrivesManager()
         {
             EnumerateDrives();
+
+            cloudProviderController = new CloudProviderController();
```

Comment:
This should be called before EnumerateDrives right? Since you call GetVirtualDrivesListAsync which uses it inside EnumerateDrives

Reply:
Yeah, your right. But it is still working...

Reply:
Magic of `async void` methods